test/integration/targets/inventory_kubevirt_conformance/constraints.txt

```
@@ -1,3 +1,4 @@
 setuptools < 45 ; python_version <= '2.7' # setuptools 45 and later require python 3.5 or later
 ruamel.yaml < 0.17 ; python_version <= '2.7'  # ruamel.yaml 0.17 and later are not tested with Python 2.7
 ruamel.yaml.clib <= 0.2.2 ; python_version < '3.5'  # ruamel.yaml.clib 0.2.3 and later requires Python 3.5 or later
+kubernetes < 25.3.0
```

OK, in that case I'd just include a generic comment explaining that. Perhaps:

```suggestion
kubernetes < 25.3.0  # newer versions no longer pass tests
```